OpenSim/Simulation/Model/Marker.h

```diff
@@ -57,24 +57,31 @@ class Body;
 //=============================================================================
 // METHODS
 //=============================================================================
-    //--------------------------------------------------------------------------
-    // CONSTRUCTION
-    //--------------------------------------------------------------------------
 public:
     Marker();
     virtual ~Marker();
+    /** Convenience method to get the 'parent_frame' Socket's connectee_name */
+    const std::string& getParentFrameName() const;
 
-    const std::string& getFrameName() const;
+    /** Convenience method to set the 'parent_frame' Socket's connectee_name
+        and to establish the connection during finalizeConnections(). */
+    void setParentFrameName(const std::string& aName);
```

Comment:
The wording can be made clearer by saying explicitly that no connection is established by this call. You need to call finalizeConnections to establish connection or something along these lines.

Reply:
Will do. Thanks.